openfl-workspace/tf_cnn_mnist/plan/plan.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) 2020-2024 Intel Corporation
+# Licensed subject to the terms of the separately executed evaluation license agreement between Intel Corporation and you.
+
+aggregator :
+  defaults : plan/defaults/aggregator.yaml
+  template : openfl.component.Aggregator
+  settings :
+    init_state_path : save/tf_cnn_mnist_init.pbuf
+    best_state_path : save/tf_cnn_mnist_best.pbuf
+    last_state_path : save/tf_cnn_mnist_last.pbuf
```

Comment:
Redundant naming. It is OK to save it as `init_model.pbuf, best_model.pbuf, last_model.pbuf`.

These are saved within workspace directory IIRC, so the user already knows which experiment it belongs to

Reply:
Thanks! Fixed for TF workspaces. Will need a separate PR for other workspaces outside the scope of this PR